test/runtests.jl

```diff
@@ -12,46 +12,58 @@ using FilePathsBase.TestPaths
 using UUIDs: uuid4
 using JSON3
 
+const AWSS3_TESTSETS = split(get(ENV, "AWSS3_TESTSETS", "MinIO,S3"), ',')
+
 is_aws(config) = config isa AWSConfig
 
 # Load the test functions
 include("s3path.jl") # creates `awss3_tests(config)`
 include("awss3.jl") # creates `s3path_tests(config)`
 
 @testset "AWSS3.jl" begin
-    if VERSION >= v"1.5"
-        using Minio
-        AWS.aws_account_number(::Minio.MinioConfig) = "123"
-
-        # We run most tests under Minio. This can be done locally by those
-        # without access to the s3 bucket under which CI is performed.
-        # We then run all tests with s3 directly.
-
-        port = 9005
-        minio_server = Minio.Server([mktempdir()]; address="localhost:$port")
-
-        try
-            run(minio_server; wait=false)
-            sleep(0.5)  # give the server just a bit of time, though it is amazingly fast to start
-            config = global_aws_config(
-                MinioConfig(
-                    "http://localhost:$port"; username="minioadmin", password="minioadmin"
-                ),
+    @testset "MinIO" begin
+        if "MinIO" in AWSS3_TESTSETS && VERSION >= v"1.5"
+            using Minio
+            AWS.aws_account_number(::Minio.MinioConfig) = "123"
+
+            # We run most tests under Minio. This can be done locally by those
+            # without access to the s3 bucket under which CI is performed.
```

Comment:
Maybe links to setting up Min.IO locally?

Reply:
The code here automatically sets up a MinIO server for you so there's nothing to pre-setup here.